src/Anon.ts

```diff
@@ -26,15 +26,21 @@ function randomUid() {
     return UIDPREFIX + "777." + rando;
 };
 
+//CMHM new function to check that what is in fields is proper
+
+
+
+
 // TODO: Test that there's no personal data stored outside the "Value" for a
 // given tag. This should be the case, and we're making the assumption that the
 // user is not maliciously trying to hide data.
 export default function anonymize(dict) {
     var newDict = {};
     for(const key of Object.keys(dict)) {
         // Use default action or action specified in policy
+    //CONSIDER MOVING NEXT LINE OUTSIDE THE LOOP! (CMHM)
```

Comment:
We need to reset `rule` to the default policy for each loop iteration - otherwise, the policy rule for the previous field would be used if none were defined.